search/src/Components/upload/upload_stepper.js

```diff
@@ -299,6 +287,28 @@ class UploadStepper extends React.Component {
         }
     }
 
+    nextHandler() {
+        const step = stepsByType[this.props.upload_type][this.state.activeStep].type
+        const currentStageValid = (step) => {
+            switch (step) {
+                case "categories":
+                    return [this.state.categories_saved, this.handleUpdateCategories]
+                case "agcontext":
+                    return [this.state.coco_form_validation['agcontexts'], this.handleUploadAgcontexts]
+                case "metadata":
+                    return [this.state.coco_form_validation['metadata'], this.handleUploadMetadata]
+                case "images":
+                    return [this.state.imageReady, this.handleNext]
+                default:
+                    return [true, this.handleNext]
```

Comment:
why do coco-upload and weedcoco-upload not have any requirements before enabling Next?